### ISSUE TITLE
[Bug Report] Pressing TAB on a clearable field focuses the clear icon instead of the next field.

### ISSUE THREAD

**Issue by TheBITLINK** [vuetifyjs/vuetify#10680]
### Environment
**Vuetify Version:** 2.2.15
**Last working version:** 1.5.23
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.122
**OS:** Linux x86_64

### Steps to reproduce
 - Focus any `clearable` input field.
 - Input some text.
 - Press the `Tab` key.

### Expected Behavior
 - The next field gets focused.

### Actual Behavior
 - The clear icon gets focused.

### Reproduction Link
<a href="https://codepen.io/TheBITLINK/pen/KKpWrRX" target="_blank">https://codepen.io/TheBITLINK/pen/KKpWrRX</a>

**Comment by KaelWD** [comment]
This is intentional: #10265

**Comment by TheBITLINK** [comment]
Can't there at least be a prop or workaround to revert to the previous behavior?

I use Vuetify for a data-centric, input heavy application and this has been messing with my users' workflow.

**Comment by ricardo17coelho** [comment]
Please add a prop or workaround to revert to the previous behavior.
It doesn't make sense that the TAB key focuses on CLEAR icon.

**Comment by mkcinu** [comment]
Any reasonable workaround for this? This is really inconvenient input heavy applications.

**Comment by drewklein20** [comment]
In some cases it makes sense to have the appended icon "focus" after pressing tab within the field. The clearable prop/icon however is NOT practical for auto-focusing.

Here is a workaround for this:

`<v-text-field
        v-model="username"
        label="Username"
        filled
        tabindex="1">
          <template v-if="username != ''" v-slot:append tabindex="0">
            <v-icon  @click="username = ''">highlight_off</v-icon>
          </template>
      </v-text-field>`
      
If the next field you would like to "tab" to, set it's tabindex property to 2, and so on and so forth (incrementing the value for each subsequent field). Notice the v-slot has a tabindex set to 0 so it will be skipped.

**Comment by philly-vanilly** [comment]
If you have to make a tradeoff between handicapped users and power-users, why not offer the programmer the choice without removing the clearable button altogether?

**Comment by philly-vanilly** [comment]
@KaelWD Please re-open this issue, the UX is not good as it is

**Comment by cbkmutlu** [comment]
This is really uncomfortable UX. Should be re-open.

Temporary solution:
```js
const handleTabIndex = () => {
    const clearButtons = document.querySelectorAll(".v-field__clearable .v-icon");
    clearButtons.forEach((icon) => {
       icon.setAttribute("tabindex", "-1");
    });
};

const observer = new MutationObserver(handleTabIndex);

//...
 app.mount("#app");
 observer.observe(document.body, {
    childList: true,
    subtree: true
 });
```

Currently, we're forced to use MutationObserver to set tabindex="-1" manually, which feels like a workaround for something that could be configurable.

Recommendation 1:
```js
export const vuetify = createVuetify({
  VTextField: {
    clearable: true,
    density: "compact",
    variant: "underlined",
  }
});
```
`clearable: -1` or `clearableTabIndex: -1`

Recommendation 2:
```js
export const vuetify = createVuetify({
  VTextField: {
    clearable: true,
    density: "compact",
    variant: "underlined",
    VIcon: {
      tabIndex: -1
    }
  }
});
```
`tabIndex` attr can be added to the component

**Comment by J-Sek** [comment]
Could you share a demo on VPlay? I just made a bunch of fields with `clearable` and cannot select them with [Tab]. Curious if we can have a demo to experience the problem.

**Comment by cbkmutlu** [comment]
> Could you share a demo on VPlay? I just made a bunch of fields with `clearable` and cannot select them with [Tab]. Curious if we can have a demo to experience the problem.
> 
> **Edit**: looks like it was resolved in v3.8.7 by [5f8208f](https://github.com/vuetifyjs/vuetify/commit/5f8208f41da4cf66907f5094312c9dbfb5e67a06)

Everything is fine in version 3.8.7, I didn't notice the new version.
